examples/implementation/platform_independent/SDF_example_002/buffer_blocking_pico2.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BUFFER_BLOCKING_PICO2_H
+#define BUFFER_BLOCKING_PICO2_H
+
+#include <pico/util/queue.h>
+#include "bsp.h"
+
+// Wrap the pico2 implementation
+typedef queue_t buffer_blocking;
+typedef int token;
```

Comment:
You don't have this typedef in the headers for the PC. You defined the `typedef int token;` in `common.h` so you can probably remove it here as well.